### PR TITLE
[DOCFIX] Improve security docs

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
@bf8086

File: docs/en/advanced/Security.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This document describes the following security related features in Alluxio.
 1. [User Authentication](#authentication): 
 Alluxio filesystem will differentiate users accessing the service
 when the authentication mode is `SIMPLE` (i.e., `alluxio.security.authentication.type=SIMPLE`).
-Alluxio also supports other authentication modes like `NOSASL` which ignores the difference among users.
+Alluxio also supports other the mode `NOSASL` which ignores authentication.
```
Comment (bf8086) [inline docs/en/advanced/Security.md]: > supports other the mode `NOSASL`

 =>

> supports `NOSASL` mode

?

File: docs/en/advanced/Security.md [hunk 2]
```diff
@@ -219,54 +219,77 @@ ACLs can be managed by two ways:
...
+When Alluxio is used in a Hadoop environment, a user, or identity, can be specified for both the
+Hadoop client and the Alluxio client. Since the Hadoop client user and the Alluxio client user can
+specified independently, the users could be different from each other. The Hadoop client user may
+even be in a separate namespace from the Alluxio client user.
+
+Alluxio client-side Hadoop impersonation solves the issues when the Hadoop client user is different
+from the Alluxio client user. With this feature, the Alluxio client examines the Hadoop client user,
+and then attempts to impersonate as that Hadoop client user.
+
+For example, a Hadoop appliction can be configured to run as the Hadoop client user `foo`, but the Alluxio client user is configured to be `yarn`. This means any data interactions will be attributed
```
Comment (bf8086) [inline docs/en/advanced/Security.md]: (nit) `appliction` => `application`

File: docs/en/advanced/Security.md [hunk 3]
```diff
@@ -219,54 +219,77 @@ ACLs can be managed by two ways:
...
+client.
+
+### Exceptions
+
+The most common impersonation error applications may see is something like
+`User yarn is not configured for any impersonation. impersonationUser: foo`. This is most likely
+due to the fact that the Alluxio servers have not been configured to enable impersonation for that
+user. To fix this, the Alluxio servers must be configured to enable impersonation for the user in question (yarn in the example error message).
+
+Please read this [blog post](http://www.alluxio.com/blog/alluxio-developer-tip-why-am-i-seeing-the-error-user-yarn-is-not-configured-for-any-impersonation-impersonationuser-foo) for more tips.
```
Comment (bf8086) [inline docs/en/advanced/Security.md]: Do we have a similar doc hosted on OS site?

File: docs/en/advanced/Security.md (hunk 1 above)
Comment (gpang) [reply docs/en/advanced/Security.md]: fixed

File: docs/en/advanced/Security.md (hunk 2 above)
Comment (gpang) [reply docs/en/advanced/Security.md]: fixed

File: docs/en/advanced/Security.md (hunk 3 above)
Comment (gpang) [reply docs/en/advanced/Security.md]: Not that I know of...

**Pull request merged**